README.md

```diff
@@ -1,33 +1,37 @@
 # AMP ⚡
 
-##### Stats:
-
 [![Build Status](https://img.shields.io/travis/ampproject/amphtml/master.svg?logo=Travis%20CI&logoColor=white&style=flat-square "Build Status")](https://travis-ci.org/ampproject/amphtml/builds)
 [![GitHub Release](https://img.shields.io/github/release/ampproject/amphtml.svg?logo=GitHub&style=flat-square "GitHub Release")](https://github.com/ampproject/amphtml/releases)
 [![Contributors](https://img.shields.io/github/contributors-anon/ampproject/amphtml.svg?logo=GitHub&style=flat-square "Contributors")](https://github.com/ampproject/amphtml/graphs/contributors)
 [![Commits](https://img.shields.io/github/commit-activity/m/ampproject/amphtml.svg?logo=GitHub&style=flat-square "Commits")](https://github.com/ampproject/amphtml/pulse/monthly)
+[![Badges](https://img.shields.io/badge/badges-16-brightgreen?logo=GitHub&style=flat-square)](#)
```

Comment:
Can we use the AMP logo instead of the GitHub logo for this one? If there isn't one, maybe use a smiley instead :)

Reply:
A URL to an SVG can be provided via the `logoSvg` URL param; alternately, since the bolt is unicode, it could just be a part of the label (although in my experience that kinda throws off the kerning a smidge). Also I can't believe we're actually adding a badge-badge :laughing: